var/spack/repos/builtin/packages/rocmvalidationsuite/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Rocmvalidationsuite(CMakePackage):
+    """The ROCm Validation Suite (RVS) is a system administrators
+       and cluster manager's tool for detecting and troubleshooting
+       common problems affecting AMD GPU(s) running in a high-performance
+       computing environment, enabled using the ROCm software stack on a
+       compatible platform."""
+
+    homepage = "https://github.com/ROCm-Developer-Tools/ROCmValidationSuite"
+    url      = "https://github.com/ROCm-Developer-Tools/ROCmValidationSuite/archive/rocm-3.7.0.tar.gz"
+
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
+
+    version('3.5.0', sha256='273e67ecce7e32939341679362b649f3361a36a22fab5f64cefe94b49e6f1e46')
+    version('3.7.0', sha256='bb42d7fb7ee877b80ce53b0cd1f04b0c8301197b6777d2edddcb44732bf8c9e2')
+
+    variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
+
+    patch('001-fixes-for-rocblas-rocm-smi-install-prefix-path.patch')
+
+    depends_on('cmake@3.5:', type='build')
+    depends_on('zlib', type='link')
+
+    def setup_build_environment(self, build_env):
+        spec = self.spec
+        build_env.set("HIPCC_PATH", spec['hip'].prefix)
+
+    for ver in ['3.5.0', '3.7.0']:
+        depends_on('hip@' + ver, type='build', when='@' + ver)
+        depends_on('rocm-device-libs@' + ver, type='build', when='@' + ver)
+        depends_on('comgr@' + ver, type='build', when='@' + ver)
+        depends_on('rocclr@' + ver, type='build', when='@' + ver)
+        depends_on('hsakmt-roct@' + ver, type='build', when='@' + ver)
+        depends_on('hsa-rocr-dev@' + ver, type='link', when='@' + ver)
+        depends_on('rocminfo@' + ver, type='build', when='@' + ver)
+        depends_on('rocblas@' + ver, type='link', when='@' + ver)
+        depends_on('rocm-smi-lib@' + ver, type='build', when='@' + ver)
+
+    def cmake_args(self):
+        spec = self.spec
+        args = ['-DHIP_INC_DIR={}'.format(spec['hip'].prefix),
```

Comment:
```suggestion
        args = ['-DHIP_INC_DIR={0}'.format(spec['hip'].prefix),
```
or:
```suggestion
        args = ['-DHIP_INC_DIR=' + spec['hip'].prefix,
```
This is required for Python 2.6 support.